create_list_delete.sh

```
@@ -0,0 +1,219 @@
+        echo "$RESPONSE"
+    else
+        echo "Failed to create project."
+        exit 1
+    fi
+}
+
+# Function to create instances
+create_instances() {
+    echo "Attempting to create $NUM_INSTANCES instances..."
+    
+    PAYLOAD_TEMPLATE='''[
```

Shellcheck (0.10.0)

[warning] Did you forget to close this single quoted string?

(SC1078)